modules/compliance-supported-profiles.adoc

```diff
@@ -1,204 +1,435 @@
+|{product-rosa} with {hcp} (ROSA HCP)
+
+|===
+[.small]
+1. The  `ocp4-cis` and `ocp4-cis-node` profiles maintain the most up-to-date version of the CIS benchmark as it becomes available in the Compliance Operator. If you want to adhere to a specific version, such as CIS v1.4.0, use the `ocp4-cis-1-4` and `ocp4-cis-node-1-4` profiles.
+2. Node profiles must be used with the relevant Platform profile. For more information, see _Compliance Operator profile types_.
+3. CIS v1.4.0 is superceded by CIS v1.5.0. It is recommended to apply the latest profile to your environment.
+4. To locate the CIS {product-title} v4 Benchmark, go to  link:https://www.cisecurity.org/benchmark/kubernetes[CIS Benchmarks] and click *Download Latest CIS Benchmark*, where you can then register to download the benchmark.
+
+[id="e8-profiles_{context}"]
+== Essential Eight compliance profiles
+
+.Supported Essential Eight compliance profiles
+[cols="2,2,1,2,1,2", options="header"]
+
+|===
+|Profile
+|Profile title
+|Application
+|Industry compliance benchmark
+|Supported architectures
+|Supported platforms
 
 |ocp4-e8
 |Australian Cyber Security Centre (ACSC) Essential Eight
 |Platform
-|0.1.39+
 |link:https://www.cyber.gov.au/acsc/view-all-content/publications/hardening-linux-workstations-and-servers[ACSC Hardening Linux Workstations and Servers]
 |`x86_64`
 |
 
-|ocp4-moderate
-|NIST 800-53 Moderate-Impact Baseline for Red Hat OpenShift - Platform level
+|rhcos4-e8
+|Australian Cyber Security Centre (ACSC) Essential Eight
+|Node
+|link:https://www.cyber.gov.au/acsc/view-all-content/publications/hardening-linux-workstations-and-servers[ACSC Hardening Linux Workstations and Servers]
+|`x86_64`
+|{product-rosa} with {hcp} (ROSA HCP)
+
+|===
+
+[id="fedramp-high-profiles_{context}"]
+== FedRAMP High compliance profiles
+
+.Supported FedRAMP High compliance profiles
+[cols="2,2,1,2,1,2", options="header"]
+
+|===
+|Profile
+|Profile title
+|Application
+|Industry compliance benchmark
+|Supported architectures
+|Supported platforms
+
+|ocp4-high ^[1]^
+|NIST 800-53 High-Impact Baseline for Red Hat OpenShift - Platform level
 |Platform
-|0.1.39+
-|link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
+|link:https://csrc.nist.gov/Projects/risk-management/sp800-53-controls/release-search#!/800-53[NIST SP-800-53 Release Search]
 |`x86_64`
- `ppc64le`
- `s390x`
 |
 
-|rhcos4-e8
-|Australian Cyber Security Centre (ACSC) Essential Eight
+|ocp4-high-node ^[1]^
+|NIST 800-53 High-Impact Baseline for Red Hat OpenShift - Node level
+|Node ^[2]^
+|link:https://csrc.nist.gov/Projects/risk-management/sp800-53-controls/release-search#!/800-53[NIST SP-800-53 Release Search]
+|`x86_64`
+|{product-rosa} with {hcp} (ROSA HCP)
+
+|ocp4-high-node-rev-4
+|NIST 800-53 High-Impact Baseline for Red Hat OpenShift - Node level
+|Node ^[2]^
+|link:https://csrc.nist.gov/Projects/risk-management/sp800-53-controls/release-search#!/800-53[NIST SP-800-53 Release Search]
+|`x86_64`
```

Comment:
Rosa hcp is supported for this ocp4-high-node-rev-4:
{product-rosa} with {hcp} (ROSA HCP)